ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
So the solution is randomly increase version of dependencies with each tested Fedora version, rather than require only the minimal needed version? I don't think that this is the way. And I doesn't like the idea to have unsynced requires between specfile and setup.py